sbc_compassion/views/correspondence_view.xml

```diff
@@ -69,6 +69,7 @@
                                 <field name="communication_type_ids" widget="many2many_checkboxes"/>
                             </group>
                             <group>
+                                <field name="redactor_id" string="Redactor name"/>
```

Comment:
put the field after `relationship` and make it visible only if `relationship` is set to `Encourager`